W3ChampionsStatisticService/Ladder/LadderController.cs

```diff
@@ -33,13 +33,17 @@ public class LadderController : ControllerBase
         public async Task<IActionResult> SearchPlayer(string searchFor, int season, GateWay gateWay = GateWay.Europe, GameMode
         gameMode = GameMode.GM_1v1)
         {
-            var playerRanks = await _rankRepository.SearchPlayerOfLeague(searchFor, season, gateWay, gameMode);
+            System.Collections.Generic.List<Rank> playerRanks;
```

Comment:
i would just make a normal import

Reply:
Ah yea, that was a leftover from when I had a conditional assignment. I didnt switch it back.